mypy/modulefinder.py

```diff
@@ -684,12 +684,22 @@ def matches_exclude(
     if fscache.isdir(subpath):
         subpath_str += "/"
     for exclude in excludes:
-        if re.search(exclude, subpath_str):
-            if verbose:
-                print(
-                    f"TRACE: Excluding {subpath_str} (matches pattern {exclude})", file=sys.stderr
-                )
-            return True
+        try:
+            if re.search(exclude, subpath_str):
+                if verbose:
+                    print(
+                        f"TRACE: Excluding {subpath_str} (matches pattern {exclude})",
+                        file=sys.stderr,
+                    )
+                return True
+        except re.error as e:
+            print(f"error: The exclude {exclude} is an invalid regular expression, because:", e)
```

Comment:
Shouldn't these go to stderr?

Reply:
I was following the lead of `error: --custom-typeshed-dir does not point to a valid typeshed`, which prints to stdout, as the most analogous thing to what I was doing, but there are also a lot of things in this file that print to stderr. Maybe the '-custom-typeshed-dir' thing is wrong to print to stdout, even.